test/account.test.js

```diff
@@ -28,12 +34,29 @@ test('create account and then view account returns the created account', async (
     await accountCreator.createAccount(newAccountName, newAccountPublicKey);
     const newAccount = new nearlib.Account(connection, newAccountName);
     const state = await newAccount.state();
-    const expectedState = { nonce: 0, account_id: newAccountName, amount: BigInt(279303400), code_hash: 'GKot5hBsd81kMupNCXHaqbhv3huEbxAFMLnpcX2hniwn', public_keys: state.public_keys, stake: BigInt(0)};
+    const expectedState = { nonce: 0, account_id: newAccountName, amount: INITIAL_BALANCE, code_hash: 'GKot5hBsd81kMupNCXHaqbhv3huEbxAFMLnpcX2hniwn', public_keys: state.public_keys, stake: BigInt(0)};
     expect(state).toEqual(expectedState);
 });
 
+test('send money', async() => {
+    const sender = await testCreateAccount();
+    const receiver = await testCreateAccount();
+    await sender.sendMoney(receiver.accountId, BigInt(10000));
+    await receiver.fetchState();
+    const state = await receiver.state();
+    expect(state.amount).toEqual(INITIAL_BALANCE + BigInt(10000));
+});
+
 // Generate some unique string with a given prefix using the alice nonce.
 const generateUniqueString = async (prefix) => {
     let status = await masterAccount.state();
     return prefix + status.nonce;
 };
+
+async function testCreateAccount() {
```

Comment:
1) `s/testCreateAccount/createAccount/`
2) I think there should be ability to override default values (like amount) to make it more flexible, i.e. similar to this https://github.com/nearprotocol/nearlib/blob/master/test/integration.test.js#L175

Reply:
Might also be worth looking into some of the test object factory modules:
https://www.npmtrends.com/factory-girl-vs-factory-js-vs-fixture-factory-vs-rosie

Reply:
Hm, not sure about the factory stuff - if it's test fixture stuff can do separately.